epub_test.go

```diff
@@ -18,6 +17,7 @@ import (
 
 	"github.com/bmaupin/go-epub/internal/storage"
 	"github.com/gofrs/uuid"
+	"github.com/stretchr/testify/require"
```

Comment:
I would prefer not to add new dependencies to the project if they don't provide any major benefit. In this case, it seems like it just avoids checking some error codes